rust/agama-cli/src/logs.rs

```diff
@@ -58,17 +74,14 @@ pub async fn run(subcommand: LogsCommands) -> anyhow::Result<()> {
             // feed internal options structure by what was received from user
             // for now we always use / add defaults if any
             let destination = parse_destination(destination)?;
-            let options = LogOptions {
-                verbose,
-                destination,
-                ..Default::default()
-            };
 
-            Ok(store(options)?)
+            // TODO: deal with return values
+            let destination = client.store(destination.as_path()).await.unwrap();
+            showln(verbose, format!("{:?}", destination).as_str());
```

Comment:
I do not think a "verbose" flag, in this case, is that useful in this context. I would always write the path, and that's it.

Reply:
no problem ... I'll remove the verbose flag.